src/Symfony/Component/Mailer/Transport/SendmailTransportFactory.php

```diff
@@ -21,7 +21,7 @@ final class SendmailTransportFactory extends AbstractTransportFactory
     public function create(Dsn $dsn): TransportInterface
     {
         if ('sendmail+smtp' === $dsn->getScheme() || 'sendmail' === $dsn->getScheme()) {
-            return new SendmailTransport(null, $this->dispatcher, $this->logger);
+            return new SendmailTransport($dsn->getOption('command'), $this->dispatcher, $this->logger);
```

Comment:
This is going to be covered by #36131

Reply:
Hm... but #36131 still does not allow to set the command for the `SendmailTransport`. Shouldn't this be implemtend regardless of #36131?